modules/persistent-storage-csi-azure-file-cross-sub-dynamic-provisioning-procedure.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-provisioning-procedure_{context}"]
+= Dynamic provisioning across subscriptions for Azure File
+
+.Prerequisites
+* Installed Azure cluster with the service principal or managed identity as an Azure identity in one subscription (call it Subscription A)
+
+* Access to another subscription (call it Subscription B) in the same tenant with the cluster
```

Comment:
same tenant "as" the cluster?

This subscription (B) should be the one with the storage IIUC, better call it out.

Reply:
That would be better.

Reply:
Done.